doc/nrf/ug_thread_ot_memory.rst

```diff
@@ -20,8 +20,9 @@ The memory requirement tables list flash memory and RAM requirements for the :re
 The values change depending on device type and hardware platform.
 Moreover, take into account the following considerations:
 
-* The sample was compiled using the default :file:`prj.conf` plus :file:`harness\overlay-cert.conf`  and :file:`overlay-multiprotocol.conf` overlay files, and the corresponding :kconfig:option:`OPENTHREAD_NORDIC_LIBRARY`, :kconfig:option:`OPENTHREAD_THREAD_VERSION` choices.
-* The single protocol sample was configured by setting :kconfig:option:`CONFIG_BT` to ``n``.
+* The sample was compiled using the default :file:`prj.conf` and the corresponding :kconfig:option:`OPENTHREAD_NORDIC_LIBRARY`, :kconfig:option:`OPENTHREAD_THREAD_VERSION` choices.
+* The sample was configured by setting :kconfig:option:`CONFIG_ASSERT` to ``n``.
```

Comment:
```suggestion
* The sample was compiled using:
   
   * The default :file:`prj.conf` and the corresponding :kconfig:option:`OPENTHREAD_NORDIC_LIBRARY`, :kconfig:option:`OPENTHREAD_THREAD_VERSION` choices.
   * The :kconfig:option:`CONFIG_ASSERT` Kconfig option set to ``n``.

```
This doesn't use the CONFIG_LOG set to `n` that was used for certification?

Reply:
Not currently. Maybe we could think about unifying configurations.